usr.sbin/daemon/daemon.c

```diff
@@ -693,12 +693,15 @@ setup_signals(struct daemon_state *state)
 		daemon_terminate(state, 1);
 	}
 
-	/* Setup SIGHUP */
```

Comment:
This looks like it changes behavior. Why is it OK to just return here rather than just call daemon_terminate?